Assets/Scripts/Player.cs

```diff
@@ -85,8 +85,10 @@ private void CheckStatus()
     {
         if (CurrentHealth <= 0)
         {
+            PlayerPrefs.SetInt("currentScene", SceneManager.GetActiveScene().buildIndex);
```

Comment:
Hai să nu hardcodăm cheile. Dacă te uiți pe main, avem clasă în Constants/PlayerPrefsKeys în care pune stringurile astea ca să nu existe riscul să le scriem greșit.